### ISSUE TITLE
Add a simple example to the README

### ISSUE THREAD

**Issue by philpax** [thebashpotato/egui-aesthetix#11]
Hiya!

Thanks for the crate; I've successfully spruced up my `egui` UI with it 🙂 

That being said, it took a bit of looking around to figure out how to use it; I had to look through `egui-dashboard-template` to find the relevant code, and found that a bit difficult as it doesn't document where things are.

To help with that, I'd suggest:
- adding a simple example of how to use the crate in the README - this can just be a few lines where the trait's imported and `set_style` is called with `Aesthetix::custom_style` on one of the styles
- indicating that `egui-dashboard-template` is a more complete example with runtime theme selection
- documenting `egui-dashboard-template` further so that it's obvious where a developer might look to find `egui-aesthetix`-related code

**Comment by Resonanz** [comment]
Yeah, the example code is way to complicated. A simple example would be super helpful :-)

**Comment by philpax** [comment]
For others on this journey, it basically just boils down to this (where `ctx` is your `egui::Context`):
```rust
ctx.set_style(Arc::new(egui_aesthetix::themes::NordDark.custom_style()));
```

**Comment by thebashpotato** [comment]
Thanks for the feedback, I'm starting on refactoring the code base today, and adding examples etc.

**Comment by saturn77** [comment]
> For others on this journey, it basically just boils down to this (where `ctx` is your `egui::Context`):
> 
> ```rust
> ctx.set_style(Arc::new(egui_aesthetix::themes::NordDark.custom_style()));
> ```

@philpax Thanks for this example, was looking for the single one line instantiation of a theme. Definitely would suggest adding this to the examples.
